src/bindings/python/tests/test_inference_engine/test_input_node.py

```diff
@@ -0,0 +1,123 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+
+from ..conftest import model_path
+from openvino.runtime import Input, Shape, PartialShape, Type, Parameter, \
+    PyRTMap
```

Comment:
I don't think this is a good name for the class from public API, especially, if to look at the previous imports. How about "RuntimeInfo"?

Reply:
@nkogteva, @ilya-lavrenov, what do you think about that kind of renaming.

Reply:
@AlexKoff88 I have no objections if it is easily can be renamed

Reply:
In the C++ this structure is named as `RTMap`, why not to keep the same name?

Reply:
@AlexKoff88 @ilya-lavrenov is it possible to agree upon one solution so this PR can be closed? Or if you need additional time to discuss it further, could it be done offline? So this ticket can be closed and if needed new one with the changes of the naming will be opened?

Reply:
@bszmelcz, I suggest renaming `PyRTMap` to `RTMap` to make the names consistent between C++ and Python. Any further renamings should be done in a separate PR and ratified before.

Reply:
@AlexKoff88 renamed